1. Created a Sort Kata 2. Added Sort topic titles to be covered in th…

yocodigo opened the pull request:
…e guide md 3. Created the Sort-katas.md file and added initial outline. I also included an example problem that is solved using one of the katas--merge sort.

Commit:
1. Created a Sort Kata 2. Added Sort topic titles to be covered in the guide md 3. Created the Sort-katas.md file and added initial outline. I also included an example problem that is solved using one of the katas--merge sort.

Trewaters commented:
This is a good start. I look forward to seeing the sorts as they get added.

I would say that since you will be solving them with JavaScript code the sort folder should fall under the JavaScript folder like so Kata > JavaScript > Sort > ...

I was on the fence about where to place this though because sort is an algorithm thing which is universal to coding.

Oh and...I will pull it in once you have completed the sorts to your satisfaction.

yocodigo commented:
I will move the Sort folder under the JavaScript folder and I agree as far as getting the commit pulled once I'm completely done with the guide and the kata.

Commit:
UPDATE

1. I moved the Sort directory and its files under the JavaScript directory, 2. Added reference links to each sort type in the Sort-Guide.md file, 3. Added a Bubble sort example into the Sort-katas.md file.

Trewaters commented:
Day #2 looking good!
